torch/csrc/jit/passes/quantization.cpp

```diff
@@ -694,6 +694,29 @@ bool isWeightOfConvOrLinear(Value* v) {
       v,
       AtenFuncArgs({{"conv2d", 1}, {"linear", 1}}),
       CallFuncArgs({{"linear", 2}}));
+
+  return result;
+}
+
+bool checkValueIsWeight(Module& module, Graph* graph, Value* v) {
+  if (isWeightOfConvOrLinear(v)) {
+    return true;
+  }
+  bool result = false;
```

Comment:
I think we want also make sure that all uses give the same result for this check, you can use a similar pattern as: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization.cpp#L983 to do that